optashift-employee-rostering-gwtui/pom.xml

```diff
@@ -128,6 +135,9 @@
         <artifactId>gwt-maven-plugin</artifactId>
         <configuration>
           <moduleName>org.optaplanner.openshift.employeerostering.gwtui.WorkerRosteringWebapp</moduleName>
+          <includes>
+            <include>**/*Test.java</include>
```

Comment:
Having an include means that everything that is not explicitly included is excluded. Is that intended?

1) It seems weird to me that we want to include *Test.java, as we might have test helper classes in src/test/java.

2) I would except the tbroyer plugin to just do the right thing. Configuration by exception. Ideally, we don't need to do any includes, and we just follow the tbroyer plugin conventions.

Reply:
@ge0ffrey it due to how the gwt plugin run tests; by default, it only runs suites (see https://tbroyer.github.io/gwt-maven-plugin/test-mojo.html#includes and https://github.com/tbroyer/gwt-maven-plugin/issues/56). Some of these tests require different runners (GwtMockitoRunner), so I am not sure if it will work when run as a Suite.

Reply:
tbroyer's choice to call this just "includes" is confusing. Only the goal gwt:test has it currently, but if other goals ever get an includes element (for example which sources to include to compile), this is going to create havoc.

Can we confine this includes configuration for the goal gwt:test? Something like:
     <execution>
                 <goal>gwt:test</>
                 <configuration>
                       <includes>...</></></>